neuralmonkey/encoders/attentive.py

```diff
@@ -0,0 +1,25 @@
+# pylint: disable=too-few-public-methods
+class Attentive(object):
+    def __init__(self, attention_type, **kwargs):
+        self._attention_type = attention_type
+        self._attention_kwargs = kwargs
+
+        assert hasattr(self, "name")
+        assert hasattr(self, "_padding")
+        assert hasattr(self, "_attention_tensor")
```

Comment:
Ač je to asi OK, nepřijde mi jako nejlepší věc diktovat z předka místo v `__init__` metodě potomků, ze kterýho se bude volat `super`.. Navíc mi přijde, že by to asi správně mělo bejt opačně, že by se `super` měl volat jako první command (á la Java). Nejde to tomu vrazit jako argument konstruktoru, když to potřebuje?